source/Pacioli/Pacioli.WebApi/Models/TokenGrantModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Pacioli.WebApi.Models
+{
+    public sealed class TokenGrantModel
```

Comment:
Why is this one sealed? I am just curious

Reply:
It was just a habit that I picked up from a book that I read. I removed it.